### ISSUE TITLE
[django] Add support django 2.2

### ISSUE THREAD

**Issue by nemesisdesign** [openwisp/django-netjsonconfig#108]
Add support for django 2.2 (only run tests on python 3.6).

Some automated tests fail with django 2.2 so it will be necessary to do some work there.

**Comment by atmishra** [comment]
I would like to work on this. Can I take this?

**Comment by NoumbissiValere** [comment]
@atmishra it looks like @EdgeKing810  has solved the issue. 
Please can you look for another issue?

**Comment by IAmAMissingColon** [comment]
Hello I think this issue has been solved in #132 as well

**Comment by nemesisdesign** [comment]
@IAmAMissingColon we need to add support for django 2.2 now and release before we can merge #132, if you want you can help with this.

**Comment by IAmAMissingColon** [comment]
Yeah, I already claimed the task :smile:

**Comment by IAmAMissingColon** [comment]
Hello !

* Pytz 
* jsonschema
* jinja2
* OpenSSL

the above dependencies needs to be installed/upgraded while enabling support for Django2.2

any suggestion ?

**Comment by IAmAMissingColon** [comment]
errors I faced:

1. error: `django-model-utils 4.0.0 is installed but django-model-utils<3.3.0,>=3.1.2 is required by {'openwisp-utils'}`
2. `django-x509 0.5 requires cryptography<2.9.0,>=2.4.0, which is not installed.`
3. `django-x509 0.5 requires pyopenssl<20.0.0,>=17.5.0, which is not installed.`

after running `./manage.py migrate` 

1. ModuleNotFoundError: `No module named 'jsonschema'`

after installing jsonschema (pip install jsonschema)

2. ModuleNotFoundError: `No module named 'jinja2'(pip install jinja2)`

after installing jinja2

3. ModuleNotFoundError: `No module named 'OpenSSL'`

after installing OpenSSL (pip install pyOpenSSL)

./manage.py migrate run without giving any error. 
./manage.py runserver runs without any error.

but unfortunately tests fails same as @EdgeKing810

**Comment by nemesisdesign** [comment]
It is stated in the issue description:

> Some automated tests fail with django 2.2 so it will be necessary to do some work there.


You have to research and find out why those tests are failing and fix them on django 2.2.

**Comment by nemesisdesign** [comment]
It is stated in the issue description:

> Some automated tests fail with django 2.2 so it will be necessary to do some work there.

You have to research and find out why those tests are failing and fix them on django 2.2.
